tempodb/backend/v1/v1.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package backend.v1;
+
+import "pkg/tempopb/tempo.proto";
+
+import "google/protobuf/timestamp.proto";
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+
+option go_package = "github.com/grafana/tempo/tempodb/backend/v1";
+
+message BlockMeta {
+    string version = 1;
+    string block_id = 2;
+    bytes min_id = 3;
+    bytes max_id = 4;
+    string tenant_id = 5;
+    google.protobuf.Timestamp start_time = 6[(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+    google.protobuf.Timestamp end_time = 7[(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+    int32 total_objects = 8;
+    uint64 size = 9;
+    uint32 compaction_level = 10;
+    int32 encoding = 11;
```

Comment:
Could this be resized to uint32?

Reply:
it can be negative?

Reply:
I think we can resize yes.  I was matching what was in BlockMeta currently, or as close as I could.

Reply:
I've pushed a commit for this.